python/cuml/test/test_coordinate_descent.py

```diff
@@ -86,7 +80,11 @@ def test_lasso(datatype, X_type, lr, algorithm,
         cu_lasso.fit(X_train, y_train)
         cu_predict = cu_lasso.predict(X_test)
 
-    if nrows != 500000:
+    if nrows < 500000:
+        sk_lasso = Lasso(alpha=np.array([lr]), fit_intercept=True,
```

Comment:
small thing but I'd use a different variable name instead of "lr" since we use it for "learning rate" mostly.